mautic/mautic
# [UX] Create a tooltip for "Placement" to explain the behavior of the bar (focus item) when it is placed at the bottom of the page

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This update introduces a tooltip for the “Positioning” feature, elucidating the behavior of the focus item when placed at the bottom of the page. It clarifies that with the sticky option disabled, the bar will appear at the end of the page content. However, if the sticky option is enabled, the bar will remain at the bottom of the user’s viewport as they scroll. This enhancement ensures users have a clear understanding of how the bar will function and appear on their page, allowing for more precise design control.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Focus Items
3. Create a new
4. On Style, choose Bar
5. Hover over Placement

![image](https://github.com/mautic/mautic/assets/116097999/f61cb6ff-d16b-4a32-9e85-55cab9528aa1)
